CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ## Master
 
+* Add Support from Bitbucket Server
```

Comment:
Nit: Mind adding your name after this like the other entries? So we know who to thank for this lovely work 😃